app/routes/chatbot.js

```diff
@@ -13,13 +12,24 @@ const phoenix = require('../../lib/phoenix');
 const stathat = require('../../lib/stathat');
 const ClosedCampaignError = require('../exceptions/ClosedCampaignError');
 
+// config modules
+const requiredParamsConf = require('../../config/middleware/chatbot/required-params');
+const requestContextConf = require('../../config/middleware/chatbot/request-context');
```

Comment:
I definitely like where Conversation is headed. This also handles generally whatever the incoming message is, whether it be only a keyword, or only an image (or combos).. spitballing ideas here:
* `incoming-message-params`
* `user-incoming-message` (could group nicely with `user-get`, `user-create`)